frontend/src/components/common/QuestionCard.tsx

```diff
@@ -0,0 +1,121 @@
+import React from "react";
+import {
+  Box,
+  Container,
+  Flex,
+  Image,
+  Spacer,
+  HStack,
+  Tag,
+  Text,
+} from "@chakra-ui/react";
+
+type QuestionCardProps = {
+  date: Date;
+  questionTitle: string;
+  image: string;
+  text: string;
+  tags: string[];
+};
+
+const QuestionCard = ({
+  date,
+  questionTitle,
+  image,
+  text,
+  tags,
+}: QuestionCardProps): React.ReactElement => {
+  return (
+    <Container
+      borderRadius="22px"
+      padding="24px"
+      maxWidth="xl"
+      background="#fff"
+    >
+      <Flex>
+        <Text fontSize="16px" lineHeight="21px" align="left">
+          {`${date.getFullYear()} / ${String(date.getMonth() + 1).padStart(
+            2,
+            "0",
+          )} / ${String(date.getDate()).padStart(2, "0")}`}
+        </Text>
+        <Spacer />
+        <HStack display={["none", "block"]}>
+          <Text
```

Comment:
maybe factor out this code to avoid duplicating again below

Reply:
now that i think about it, maybe you can have the two buttons in one Box and the rest of the text/tags in another Box and wrap them with a `<Stack>` and use [responsive direction](https://chakra-ui.com/docs/components/stack#responsive-direction). not sure if that will work but might make the code a little cleaner (and also the text will not run under the buttons like it is in figma)